src/components/Footer.tsx

```diff
@@ -4,6 +4,13 @@ const Footer = () => (
   <div className="flex flex-row justify-between text-xs font-monospace opacity-75">
     <span>© Cap Finance</span>
     <div className="flex flex-row space-x-12">
+      <a
```

Comment:
I think we can only have up to three links in the footer, after that it overflows the page on mobile.

Reply:
fair enough, removed